### PR TITLE
docs: 增加天使轮融资

### PR DESCRIPTION
**Pull request by juzhiyuan**

### CODE REVIEW

**Comment by juzhiyuan** [comment]
ping @moonming

File: data/about.json [hunk 1]
```diff
@@ -55,6 +59,10 @@
       }
     ],
     "en-US": [
+      {
+        "date": "December 2020",
+        "desc": "Received million dollars angel round funding from ZhenFund"
```
Comment (moonming) [inline data/about.json:64]: APISEVEN Raises $1 Million in Series Angel Funding Led by ZhenFund

**Review by moonming — CHANGES_REQUESTED** [review]

File: data/about.json (hunk 1 above)
Comment (LiteSun) [reply data/about.json]: updated.

**Comment by juzhiyuan** [comment]
@moonming updated

**Review by moonming — APPROVED** [review]

**Pull request merged**